src/ofxstatement_schwab_json/plugin.py

```diff
@@ -74,13 +75,19 @@ def import_lines(self, transactions):
                 self.add_buy_line(id, date, tran)
             elif action == "ADR Mgmt Fee":
                 self.add_bank_line(id, date, "SRVCHG", tran)
+            elif action == "Advisor Fee":
+                self.add_bank_line(id, date, "XFER", tran)
             elif len(tran["Symbol"]) > 0 and (
                 action == "Journaled Shares"
                 or action == "Spin-off"
                 or action == "Stock Split"
                 or action == "Security Transfer"
             ):
                 self.add_transfer_line(id, date, tran)
+            elif len(tran["Symbol"]) > 0 and (
+                action == "Journal"
```

Comment:
Should we just add another `or action == "Journal"` to the elif above (line 84)?